docs/tomcat.md

```diff
@@ -0,0 +1,49 @@
+
+## Configuration
+
+| Field                 | Default            | Description |
+| ---                   | ---                | ---         |
+| `type`                | required           | Must be `tomcat`. |
+| `endpoint`            | `localhost:8050`   | The [JMX Service URL](https://docs.oracle.com/javase/8/docs/api/javax/management/remote/JMXServiceURL.html) or host and port used to construct the Service URL. Must be in the form of `service:jmx:<protocol>:<sap>` or `host:port`. Values in `host:port` form will be used to create a Service URL of `service:jmx:rmi:///jndi/rmi://<host>:<port>/jmxrmi`. |
+| `collect_jvm_metrics` | true               | Should the set of support [JVM metrics](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/docs/jvm.md#metrics) also be collected |
+| `username`            | not set by default | The configured username if JMX is configured to require authentication. |
+| `password`            | not set by default | The configured password if JMX is configured to require authentication. |
+| `collection_interval` | `60s`              | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    tomcat_metrics:
+      type: tomcat
+      collection_interval: 60s
+  service:
+    pipelines:
+      tomcat_pipeline:
+        receivers:
+          - tomcat_metrics
+```
+
+## Metrics
+In addition to Tomcat specific metrics, by default Tomcat will also report [JVM metrics](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/docs/jvm.md#metrics)
+
+| Metric                                               | Data Type      | Unit        | Labels                         | Description |
+| ---                                                  | ---            | ---         | ---                            | ---         | 
+| workload.googleapis.com/tomcat.sessions              | Gauge          | sessions    |                                | The number of active sessions. |
+| workload.googleapis.com/tomcat.errors                | Cumulative     | errors      | proto_handler                  | The number of errors encountered. |
+| workload.googleapis.com/tomcat.processing_time       | Cumulative     | ms          | proto_handler                  | The total processing time. |
+| workload.googleapis.com/tomcat.traffic               | Cumulative     | by          | proto_handler, direction       | The number of bytes transmitted and received. |
+| workload.googleapis.com/tomcat.threads               | Gauge          | threads     | proto_handler, state           | The number of threads. |
+| workload.googleapis.com/tomcat.max_time              | Cumulative     | ms          | proto_handler                  | Maximum time to process a request. |
```

Comment:
Should `tomcat.max_time` be a Gauge instead? (I tried to find a reference in the scoping doc, but this metrics does not exist there).

Reply:
It was an additional metric otel wanted and went back and checked and you are correct in it being a gauge. Opened a PR downstream to update it but if we don't want this additional metric we can also just remove it.

Reply:
Updated the metric as the downstream change got merged already.

Reply:
Great